.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+on:
+  - push
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        node-version:
+          - '10'
```

Comment:
nitpick: could drop 10 here, LTS maintain will end April 2021. Also this being more of a client-side than server-side module I doubt versions have any effect at all here other than test the implementation rigor of the devDeps.